src/Datadog.Trace.ClrProfiler.Native/logging.cpp

```diff
@@ -0,0 +1,15 @@
+#include <spdlog/sinks/rotating_file_sink.h>
+#include <spdlog/sinks/stdout_sinks.h>
+
+#include "logging.h"
+
+namespace trace {
+std::shared_ptr<spdlog::logger> GetLogger() {
+  auto logger = spdlog::get("profiler");
+  if (logger == nullptr) {
+    logger = spdlog::rotating_logger_mt("profiler", "C:\\Temp\\Profiler.log",
```

Comment:
Should we change this path to `C:\ProgramData\Datadog\logs\dotnet-profiler.log`?